power.mss

```diff
@@ -1,9 +1,10 @@
-@power-line-color: #888;
```

Comment:
This was intended to have two distinct colors regarding power : live color for elements which transmit power and another for grounded supports

Reply:
> two distinct colors regarding power

It's not necessary to change this line to implement that. I find "power-live-color" more confusing than "power-line-color", especially since it is only used for `power=line`, `power=minor_line` and `power=cable` features which are all types of power lines, and because a feature mapped as `power=line` may be disused, yet would not cease to be a `power=line`.